src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -302,7 +303,12 @@ private void BuildPreprocessorStateChangeMap()
         /// <summary>
         /// Creates a new syntax tree from a syntax node.
         /// </summary>
-        public static SyntaxTree Create(CSharpSyntaxNode root, CSharpParseOptions options = null, string path = "", Encoding encoding = null)
+        public static SyntaxTree Create(
```

Comment:
>Create [](start = 33, length = 6)

This looks like a breaking change - modification of a public API. We probably should add a new overload, the same way how we add constructors for CompilationOptions for example.